src/routes/usersRouter.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+
+const {
+  addUser,
+  getUser,
+  updateUser,
+  deleteUser,
+  getRecords,
+  deleteRecords,
+} = require('../controller/userController');
+
+//users == base path
+router.route('/signup').post(addUser);
+router.route('/login').get(getUser);
+router.route('/:id').put(updateUser).delete(deleteUser);
+router.route('/dashboard').get(getRecords).delete(deleteRecords);
+
```

Comment:
Right now `/users/dashboard` would be a general route, so it wouldn't depend on the user this is what you want? Or would we like to be more specific so each user has dashboard route?

Reply:
Ah you are right. 
I guess every user has his own dashboard. We could set it in the userprofile/dashboard or /:id/dashboard. /dashboard would be not correct cause the user should be logged in to see it. Do you agree or what do you think?

Reply:
Yes we could make it `/:id/dashboard` but it doesn't have to be. We could protect the route. So we can have `/dashboard` because it looks cleaner and then we make the calls to dashboard checking if the user is there. Usually you don't nest the routes I think. Check vercel.com for instance.

Reply:
This how there route looks https://vercel.com/dashboard, Then we have something checking, if there is not user logged in we send him to the login page.

Reply:
then lets protect it and use it ?

Reply:
Yes, we can have just `/dashboard` and we check if the user is logged in in the front-end and in the back end. For the back-end don't worry now. We don'T know how to do it for now, we could check later.